src/components/users/signin/ko/runtime/signin.html

```diff
@@ -14,6 +14,10 @@
                 data-bind="textInput: password">
         </div>
 
+        <!-- ko if: termsEnabled && termsOfUse -->
+        <terms-of-use params="{ isConsentRequired: isConsentRequired, consented: consented, termsOfUse: termsOfUse }"></terms-of-use>
```

Comment:
I think the request was about "Complete signup" view, hence it should be only signupSocial.html. Currently, we can't enforce consent on sign-in, since it requires backend support for that, but for creating account we do support it now.